engine/src/test/scala/cromwell/CromwellTestkitSpec.scala

```diff
@@ -40,6 +41,8 @@ case class TestBackendLifecycleActorFactory(config: Config) extends BackendLifec
   }
 
   override def workflowFinalizationActorProps(): Option[Props] = None
+
+  override def engineFunctions(workflowDescriptor: BackendWorkflowDescriptor, jobKey: BackendJobDescriptorKey, configurationDescriptor: BackendConfigurationDescriptor): WdlStandardLibraryFunctions = ???
```

Comment:
Is this a good idea? Is there a simple no-op engine function we can use here instead?